content/03-reference/01-tools-and-interfaces/05-prisma-cli/02-command-reference.mdx

```diff
@@ -495,14 +495,14 @@ The `db push` command pushes the state of your Prisma schema file to the databas
 
 - You want to replicate your schema changes in other environments where there could be important data you cannot afford to lose. You can use `db push` for prototyping, but you should use migrations to commit the schema changes and apply these in your other environments.
 - You want fine-grained control over how the schema changes are executed - for example, renaming a column instead of dropping it and creating a new one.
-- You want to keep track of changes made to the database schema over time. `db push` does not create any artifacts that allow you to keep track of this information.
+- You want to keep track of changes made to the database schema over time. `db push` does not create any artifacts that allow you to keep track of these changes.
 - You want the schema changes to be reversible. You can use `db push` again to revert to the original state, but this might result in data loss and there will be no mechanism for you to orchestrate these changes.
 
 #### <inlinecode>db push</inlinecode> and Migrate (experimental)
 
 If you are already using migrations with Prisma Migrate, `db push` will still work. However, be aware of the following side effects of using both Prisma Migrate and `db push`:
 
-Prisma Migrate introspects the database to anticipate the changes needed to reach the end-state of the migration and preview these changes as DDL statements in [`README.md`](http://readme.md) files.
```

Comment:
hihi

Reply:
@janpio Are you laughing? 😂

Reply:
I am. 

What editor are you using that it made that into a link?